docs/scanners/uswds.md

```diff
@@ -1,71 +1,52 @@
-# U.S. Web Design System (USWDS) Scanner
+# U.S. Web Design System Scanner
 
 ## Summary
-A [scan](https://site-scanning.app.cloud.gov/searchUSWDS/) to check each domain for the presence or absence of a USWDS component (i.e. font, style sheet) and the version in use of USWDS.
+The [USWDS scanner](https://site-scanning.app.cloud.gov/searchUSWDS/) checks each domain for the use of [U.S. Web Design System (USWDS)](https://designsystem.digital.gov/) code and the code version.
 
-Specifically, the scan [searches for](https://github.com/18F/domain-scan/blob/tspencer/200scanner/scanners/uswds2.py#L36-L123): presence of `-usa` class in the style sheets, `uswds` in HTML, `.usa-` in HTML, `favicon-57.png` (the USWDS flag) on a website, the fonts used by USWDS (current and former versions) - `Public Sans`, `Source Sans`, and `Merriweather` — `uswds` or `uswds version` or `flavicon-57.png` in CSS. It also searches for the presence of HTML tables, although it uses those as a negative indicator that a site is using USWDS.
+Specifically, the scan [searches for](https://github.com/18F/domain-scan/blob/tspencer/200scanner/scanners/uswds2.py#L36-L123): 
 
-* USWDS adoption
-* USWDS configuration
+* Presence of `-usa` class in CSS
+* `uswds` in HTML
+* `.usa-` in HTML
```

Comment:
This is an interesting one. As far as I can tell, it's looking for the `.usa-` string in HTML. This seems be only a modest difference from the earlier check — looking for the string unconnected from the `class` string.

@timothy-spencer is this intentional?

Link: https://github.com/18F/domain-scan/blob/8223c7110d5f2bfbb8205379e09b4cea11519983/scanners/uswds2.py#L53

Reply:
Yeah, I can't remember back about why I made them separate, but I _think_ that the class.*usa thing didn't match nearly as much as .usa- did, so I thought we could break them out so that we could weight them differently?